Dockerfile

```diff
@@ -1,21 +1,21 @@
 FROM amazonlinux:2.0.20200722.0
 
 ARG STACK_VERSION=2.5.1
+ARG USER=haskell
 
-ENV LANG=C.UTF-8 \
-  LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ENV PATH="/home/$USER/.local/bin:$PATH"
```

Comment:
This way we can install things into the default Stack installation directory without doing anything weird like using `sudo` or copying things around.